docs/source/graph-construction/gs-processing/usage/aws-infra-index.rst

```diff
@@ -0,0 +1,33 @@
+================================================
+Running distributed processing jobs on AWS Infra
+================================================
+
+After successfully installing the necessary dependencies for GSProcessing and
+configuring the input parameters, you can now initiate GSProcessing jobs utilizing AWS resources.
```

Comment:
Maybe we can say "building the appropriate Docker image and pushing to the Amazon ECR" here? To run distributed jobs we don't need to install the dependencies of GSProcessing.

Reply:
That makes sense, I will move to that.